remote/remote.go

```diff
@@ -652,7 +668,7 @@ func (i *Image) Save(additionalNames ...string) error {
 	cfg.History = make([]v1.History, len(layers))
 	for i := range cfg.History {
 		cfg.History[i] = v1.History{
-			Created: v1.Time{Time: imgutil.NormalizedDateTime},
+			Created: creationTime,
```

Comment:
@ekcasey rightly [pointed out](https://buildpacks.slack.com/archives/CCP60GJLS/p1644533010840709?thread_ts=1644440749.308319&cid=CCP60GJLS) that we forgot about the daemon case - @imjasonh would you be amenable to adding that? It would go [here](https://github.com/buildpacks/imgutil/blob/14fe798ec25ae24f7225fd24cb11bf7a3e276a1d/local/local.go#L797).